### ISSUE TITLE
cann't running simple-cluster

### ISSUE THREAD

**Issue by kevindwei** [m3db/m3db-operator#77]
when i exec " kubectl apply -f example/simple-cluster.yaml"
![11](https://user-images.githubusercontent.com/20273599/51009831-ac30e100-158c-11e9-9964-20f5777aec51.PNG)

**Comment by schallert** [comment]
Hey @kevindwei, thanks for the issue and sorry for not seeing this sooner!

Can you list the zones that your cluster has nodes in, and the values of `failure-domain.beta.kubernetes.io/zone` for those nodes? The simple cluster has a [hardcoded](https://github.com/m3db/m3db-operator/blob/master/example/simple-cluster.yaml#L10-L15) list of example zones that correspond to GCP zones and would have to be modified if your nodes don't match that list.

Our current focus is clusters that have nodes in at least 3 zones to provide fault tolerant clusters. You can find a bit more about our cluster requirements in our [docs](https://operator.m3db.io/getting_started/requirements/). You can subscribe to #68 for progress on supporting single-zone clusters.

In the future we may provide a mode where the operator can auto-discover available zones, but for now we require they be hardcoded in the manifest to explicitly define the zone topology and weights.

Feel free to ping me here or in our [gitter](https://gitter.im/m3db/kubernetes) if you have further questions!

**Comment by yunstanford** [comment]
Yeah, aws is a little different, it works for me after modifying.
